### ISSUE TITLE
I would like Notepad++ to be able to be opened multiple times at the same time so that there can be multiple instances/windows by just clicking the shortcut to the program again.

### ISSUE THREAD

**Issue by odielag** [notepad-plus-plus/notepad-plus-plus#4195]
### Steps to Reproduce the Issue

1. (Open Program)
2. (Program doesn't create new program.)
3. 

### Debug Information

(original phrasing for redundancy: I would like Notepad++ to be able to be opened multiple times at the same time so that there can be multiple instances/windows by just clicking the shortcut to the program again.)

**Comment by Arffeh** [comment]
Settings - > Preferences

Multi-Instance - > Select your preferred behaviour.

**Comment by odielag** [comment]
It didn't work.

I followed your instructions to those settings and none of the three
options allowed a new window of Notepad++ to be opened when I reclicked the
icon.

**Comment by sasumner** [comment]
Sometimes we just have to decide that the provided behavior is sufficient.

**Comment by proeng** [comment]
I tried checking the always in multi instance mod, you need to restart the program before it takes effect.

**Comment by dail8859** [comment]
`Settings > Preferences > Multi-Instance` works for me as well.

**Comment by odielag** [comment]
The functionality you claim it has | it does not seem to have.

I set it to multi-instance as described, and it did not work. Whenever the icon for the program was clicked it would just maximize and minimize the program and not make any new instances.

I also uninstalled notepad++ from the windows control panel, redownloaded the install from the website about 5 minutes ago, and the functionality was still not there after setting it to Multi-Instance

It's up to you if you want to make your product better and stand the test of time.
